packages/react-component-library/src/components/Toast/partials/StyledToastLabel.tsx

```diff
@@ -4,12 +4,10 @@ import { selectors } from '@defencedigital/design-tokens'
 const { color, fontSize, spacing } = selectors
 
 export const StyledToastLabel = styled.span`
-  display: inline-flex;
```

Comment:
`display: inline-flex;` was being overridden with `display: block;`, so I've removed the flexbox rules (the alternative, removing `display: block;`, broke the positioning of the icon).